Aliases/snapclient

```diff
@@ -0,0 +1 @@
+../Formula/snapcast.rb
```

Comment:
The project is called Snapcast, there is no ambiguity. I don't think we want aliases for `snapclient` and `snapserver`.